script.js

```diff
@@ -1,7 +1,32 @@
 console.log("hello script js");
 
-var inputHappened = function(currentInput){
-  console.log( currentInput );
-  var output = "WOW TEMPERATURE";
-  return output;
-};
+var convertKelvin = function(num){
+  if (parseInt(num)) {
+    var kelvin = (num - 32) * 5/9 + 273.15;
+    console.log(kelvin);
+  } else {
+    console.log("Please use a number.");
+  }
+  return kelvin;
+}
+
+var convertCelcius = function(num){
+ if (parseInt(num)) {
+    var celcius = (num - 32) * 5/9;
+    console.log(celcius);
+  } else {
+    console.log("Please use a number.");
+  }
+  return celcius;
+}
+
+var enteredFahrenheit = function(num){
+if (parseInt(num) < 0) {
```

Comment:
Since you're using `parstInt(num)` repeatedly as conditions, you can store it in a variable, and use this variable instead of calling `parseInt(num)` 3 times here :) 

Good job!